doc/nitrocli.1

```diff
@@ -318,6 +318,37 @@ The admin PIN cannot be unblocked.
 This operation is equivalent to the unblock PIN option provided by \fBgpg\fR(1)
 (using the \fB\-\-change\-pin\fR option).
 
+.SS Extensions
+In addition to the above built-in commands, \fBnitrocli\fR supports
+user-provided functionality in the form of extensions. An extension can be any
+executable file whose filename starts with "nitrocli-" and that is discoverable
+through lookup via the \fBPATH\fR environment variable. Those executables can be
+invoked as regular subcommands (without the need of the prefix; e.g., an
+extension with the name "nitrocli-otp-cache" could be invoked as "nitrocli
+otp-cache").
+
+\fBnitrocli\fR conveys basic configuration information to any extension being
```

Comment:
The first paragraph decribes how to use the extension.  The rest of the section describes the interface between nitrocli and the extensions.  How about a separate section for the interface definition, e. .g “Writing extensions”?